server/models/video.js

```diff
@@ -540,7 +547,12 @@ function listForApi (start, count, sort, callback) {
       },
 
       this.sequelize.models.Tag
-    ]
+    ],
+    where: {
```

Comment:
You need to put these where clause in the `searchAndPopulateAuthorAndPodAndTags` method too